app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -149,6 +154,40 @@ public void setupCharts() {
 
         chart.setZoomType(ZoomType.HORIZONTAL);
 
+        //Transmitter Battery Level
+        final Sensor sensor = Sensor.currentSensor();
+        if (sensor != null && sensor.latest_battery_level != 0 && sensor.latest_battery_level <= Constants.TRANSMITTER_BATTERY_LOW) {
```

Comment:
actually should we put in a check here to never show anything if they are using dexcom share